src/react/ui-elements/FormLayout.jsx

```diff
@@ -90,13 +94,18 @@ const LabelContainer = styled.label`
 `;
 
 const TooltipContainer = styled.div`
-    margin-left: 10px;
+    margin-left: 8px;
+`;
+
+const IconQuestionCircle = styled.i`
+    color: #434343;
 `;
 
 const UlOverlay = styled.ul`
-    width: 200px;
+    width: 200;
```

Comment:
Omitting units? 
We might want to get rid of the `UlOverlay` styled component if core-ui handle the tooltip sizing.

Reply:
Oh yes, my bad for the unit.
However I can't remove the UlOverlay because this is what allows us to have the bullet points before the tooltip message.
But yes, I have to remove the width to allow core-ui to resize the tooltip as its want.

Then without width rule in UlOverlay and the new version of CoreUI it will become like this:
![image](https://user-images.githubusercontent.com/44285344/113437086-4687d400-93e6-11eb-8bc6-7991e413ccf1.png)
![image](https://user-images.githubusercontent.com/44285344/113437098-4f78a580-93e6-11eb-8a18-addbf7693f67.png)